sonarts-core/tests/rules/noNestedTemplateLiteralsRule/noNestedTemplateLiteralsRule.lint.ts

```diff
@@ -35,6 +35,22 @@ function nokNestedTaggedTemplateLiterals2(x: number) {
   //                                  ^^^^^^^^^^^^^^^^^^^ {{Extract this template literal into a dedicated statement.}}
 }
 
+function nokNestedTaggedTemplateLiteralsTernary(x: number) {
```

Comment:
@andrea-guarino-sonarsource test one thing at a time -> remove tag from this test